static/css/impala/buttons.less

```diff
@@ -298,9 +298,44 @@ button[disabled],
     color: #919497;
     .box-shadow(0 3px rgba(0, 0, 0, 0.05), 0 -4px rgba(0, 0, 0, 0.05) inset);
     text-shadow: 0 1px 0 rgba(255,255,255,.5);
+    white-space: normal;
```

Comment:
What I usually do is check (using the web dev tools) where the current styling comes for the element I'm trying to change.

In this case, opening the page in chromium and inspecting the button, I see it's coming from https://github.com/mozilla/olympia/blob/master/static/css/zamboni/zamboni.css#L496

That would be my first guess ;)